clients/config-center-client/src/main/java/org/apache/servicecomb/config/center/client/ConfigCenterManager.java

```diff
@@ -77,10 +79,15 @@ public void execute() {
           ConfigurationChangedEvent event = ConfigurationChangedEvent
               .createIncremental(configConverter.getCurrentData(), lastData);
           eventBus.post(event);
+          retryTimes=0;
         }
         startTask(new BackOffSleepTask(POLL_INTERVAL, new PollConfigurationTask(0)));
       } catch (Exception e) {
         LOGGER.error("get configurations from ConfigCenter failed, and will try again.", e);
+        ++retryTimes;
+        if(retryTimes < 3) {
+          startTask(new BackOffSleepTask(0, new PollConfigurationTask(0)));
+        }
```

Comment:
Do not need this code. 
`startTask(new BackOffSleepTask(failCount + 1, new PollConfigurationTask(failCount + 1)))` already doing retry.

Reply:
done